src/east/workspace_commands/codechecker_helpers.py

```diff
@@ -0,0 +1,178 @@
+import glob
+import os
+import plistlib as plist
+import re
+
+
+def check_for_compile_commands_json(east, compile_commands):
```

Comment:
Maybe check for the build dir first?

This way you could suggest building to the user 1st.
If the build dir exists and `compile_commands.json` does not, print the text you have here.

Reply:
Makes sense, will do that.